src/Raven.Server/Documents/PeriodicBackup/BackupTask.cs

```diff
@@ -271,6 +277,16 @@ private long GetDatabaseEtagForBackup()
             }
         }
 
+        private long GetDatabaseIndexForCompareExchange()
+        {
+            using (_serverStore.ContextPool.AllocateOperationContext(out TransactionOperationContext context))
+            using (context.OpenReadTransaction())
+            {
+                var databaseRecord = _serverStore.Cluster.ReadRawDatabase(context, _database.Name, out _);
```

Comment:
Similar method pops-up in few places. Can we create one method 'somewhere' to handle this? Maybe in `DocumentDatabase`?

Reply:
done